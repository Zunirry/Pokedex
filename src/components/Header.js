import React from 'react';
import {
    View,
    Text,
    Image,
    Dimensions,
    StyleSheet
} from 'react-native'
import logo from '../../assets/pokemon.svg.png'

const windowWidth = Dimensions.get('window').width;
const windowHeight = Dimensions.get('window').height;

const Header = () => {
    return (
        <View>
            <Image
                source={logo}
                style={styles.img}
            />
        </View>
    );
}

const styles = StyleSheet.create({

    img: {
        width: windowWidth * 0.6,
        height: 80,
    }
})


export default Header;
