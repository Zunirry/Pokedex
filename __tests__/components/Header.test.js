import React from 'react';
import Header from '../../src/components/Header'
import { create } from 'react-test-renderer'

describe('<Header />', () => {

    test('Render of Header component ', () => {
        const header = create(<Header />).toJSON()
        expect(header).toMatchSnapshot();
    })
    
});



export default Header.test;
