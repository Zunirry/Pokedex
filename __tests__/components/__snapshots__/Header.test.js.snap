// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Render of Header component  1`] = `
<View>
  <Image
    source={
      Object {
        "testUri": "../../../assets/pokemon.svg.png",
      }
    }
    style={
      Object {
        "height": 80,
        "width": 450,
      }
    }
  />
</View>
`;
